modules/SerialCommHub/tiny_modbus_rtu.cpp

```diff
@@ -296,13 +303,51 @@ int TinyModbusRTU::read_reply(uint8_t* rxbuf, int rxbuf_len) {
     return bytes_read_total;
 }
 
+std::vector<uint16_t> TinyModbusRTU::txrx(uint8_t device_address, FunctionCode function,
+                                          uint16_t first_register_address, uint16_t register_quantity,
+                                          uint16_t max_packet_size, bool wait_for_reply,
+                                          std::vector<uint16_t> request) {
+    // This only supports chunking of the read-requests.
+    std::vector<uint16_t> out;
+
+    const uint16_t register_chunk = (max_packet_size - MODBUS_MIN_REPLY_SIZE) / 2;
+    size_t written_elements = 0;
+    while (register_quantity) {
+
```

Comment:
remove empty line. did you run clang-format?

Reply:
yes - but can remove it